virt/virt-4-13-release-notes.adoc

```diff
@@ -0,0 +1,266 @@
+:_content-type: ASSEMBLY
+[id="virt-4-13-release-notes"]
+= {VirtProductName} release notes
+include::_attributes/common-attributes.adoc[]
+:context: virt-4-13-release-notes
+
+toc::[]
+
+[id="virt-4-13-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
+
+
+== About Red Hat {VirtProductName}
+
+Red Hat {VirtProductName} enables you to bring traditional virtual machines (VMs) into {product-title} where they run alongside containers, and are managed as native Kubernetes objects.
+
+{VirtProductName} is represented by the image:virt-icon.png[{VirtProductName},40,40] icon.
+
+You can use {VirtProductName} with either the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] or the xref:../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN] default Container Network Interface (CNI) network provider.
+
+Learn more about xref:../virt/about-virt.adoc#about-virt[what you can do with {VirtProductName}].
+
+Learn more about xref:../virt/virt-architecture.adoc#virt-how-virt-works_virt-architecture[{VirtProductName} architecture and deployments].
+
+xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[Prepare your cluster] for {VirtProductName}.
+
+include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
+
+
+[id="virt-guest-os"]
+=== Supported guest operating systems
+//CNV-16390 Supported guest operating systems
+To view the supported guest operating systems for {VirtProductName}, refer to link:https://access.redhat.com/articles/973163#ocpvirt[Certified Guest Operating Systems in Red Hat OpenStack Platform, Red Hat Virtualization and OpenShift Virtualization].
+
+
+[id="virt-4-13-new"]
+== New and changed features
+
+//CNV-18488 SVVP for 4.12: Ensure platform passes Windows Server Virtualization Validation Program - with RHCOS workers
+//NOTE: This is a recurring release note. Modify the existing note text below if recommended by QE.
+* OpenShift Virtualization is certified in Microsoft's Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
++
+The SVVP Certification applies to:
++
+** Red Hat Enterprise Linux CoreOS workers. In the Microsoft SVVP Catalog, they are named __Red Hat OpenShift Container Platform 4 on RHEL CoreOS 8__.
+** Intel and AMD CPUs.
+
+
+//CNV-18462 Release note: CHANGE  Retire alpha APIs for Storage
+
+//CNV-19043 Release note: CHANGE Remove rhel6 support
```

Comment:
Should CNV-19043 and CNV-18462 be moved to the **Removed features** section?

Reply:
done!